Example/Tests/ViewModels/ConnectionsViewModelSpec.swift

```diff
@@ -0,0 +1,222 @@
+//
+//  ConnectionsViewModelSpec
+//  This file is part of the Salt Edge Authenticator distribution
+//  (https://github.com/saltedge/sca-authenticator-ios)
+//  Copyright © 2021 Salt Edge Inc.
+//
+//  This program is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License as published by
+//  the Free Software Foundation, version 3 or later.
+//
+//  This program is distributed in the hope that it will be useful, but
+//  WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+//  General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program. If not, see <http://www.gnu.org/licenses/>.
+//
+//  For the additional permissions granted for Salt Edge Authenticator
+//  under Section 7 of the GNU General Public License see THIRD_PARTY_NOTICES.md
+//
+
+import Foundation
+import Quick
+import Nimble
+import UIKit
+import SEAuthenticator
+import RealmSwift
+@testable import SEAuthenticatorCore
+
+class ReachabilityManagerSpy: ReachabilityManagerProtocol {
```

Comment:
After you rename `ReachabilityManagerProtocol` to `Connectivity` it should be renamed to `MockConnectivity`. Also make it `private`.

Reply:
I am waiting for a response on the comments [here](https://github.com/saltedge/sca-authenticator-ios/pull/212#discussion_r684139711)